ydb/docs/en/core/changelog-cli.md

```diff
@@ -1,5 +1,26 @@
 # {{ ydb-short-name }} CLI changelog
 
+## Version 2.18.0 {#2-18-0}
+
+Released on December 24, 2024. To update to version **2.18.0**, select the [Downloads](downloads/index.md#ydb-cli) section.
+
+* Query plan and statistics improvements:
+  * Expression and attributes added to various operator properties (i.e. GroupBy)
+  * Per operator statistics (Rows)
+  * Statistics from Column Shards (Rows and Bytes)
```

Comment:
Clarify what those Rows and Bytes mean. Probably rows processed for operators and rows/bytes read from column shards, but I had to guess that.